server/axios_api.js

```diff
@@ -0,0 +1,62 @@
+var axios = require("axios").default;
+var HOST = "movies-tvshows-data-imdb.p.rapidapi.com";
+var API_KEY = "030611f594msh73349b7b00af6e8p16d57ejsncbf1b45a6165";
+var URL = "https://movies-tvshows-data-imdb.p.rapidapi.com/";
+
+
+
+function get_details(id) {
+  var options = {
+    method: 'GET',
+    url: URL,
+    params: {type: 'get-movie-details', imdb: id},
+    headers: {
+      'x-rapidapi-host': HOST,
+      'x-rapidapi-key': API_KEY
+    }
+  };
+  axios.request(options).then(function (response) {
+    console.log("rating:"+response.data["imdb_rating"]+"\n"+"popularity:"+response.data["popularity"]+"\n"+"vote:"+response.data["vote_count"]);
+  }).catch(function (error) {
+    console.error(error);
+  });
+
+}
+
+
+function get_id(t){
+  var options = {
+    method: 'GET',
+    url: URL,
+    params: {type: 'get-movies-by-title', title: t},
+    headers: {
+      'x-rapidapi-host': HOST,
+      'x-rapidapi-key': API_KEY
+    }
+  };
+  
+  axios.request(options).then(function (response) {
+    var results = response.data.movie_results;
+    results.forEach(function(e,i){
+      console.log(e['imdb_id'])
```

Comment:
does this api return multiple movies per title?

Reply:
yes, depending if multiple movies have the same OR similar name

Reply:
okay I think in that case, either we shall error out or just return first result.